test/test-common.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+# vim: ts=4 sw=4 ts
+
+test_id=$(echo $0 | sed -e 's/^[^a-z]*//' | sed -e 's/.sh$//')
```

Comment:
Please use shell substitutions instead. For example, "${0%.sh}" will remove the extension. See the section "Parameter Expansion" of e.g. dash for more details (for convenience: http://linux.die.net/man/1/dash).